CUSTOMIZATION.md

```diff
@@ -87,9 +87,39 @@ The header may optionally define the following:
     this line.  Note: this value must be given as a string in double
     quotes, rather than as a number.
 
-## Home Page: Schedule, Syllabus, and Setup
+## Home Page: Schedule and Syllabus
 
 You should edit the sections titled `Schedule` and `Syllabus`
 so that they show what you're actually planning to teach and when.
-You should also delete irrelevant parts of the section titled `Setup`
+
+## Home Page: Setup
+
+You should irrelevant parts of the section titled `Setup`
 so that your learners don't try to install software that they won't need.
```

Comment:
"You should irrelevant parts of the section titled `Setup` so that your learners don't try to install software that they won't need."
->
"You should delete the pieces of the `Setup` section related to software you will not be using in your workshop, so that learners don't spend time installing software they don't need."